Article/Article.js

```diff
@@ -1,20 +1,29 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
+
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.lastElementChild;
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "Expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener("click", () => {
```

Comment:
remember that you can omit curly braces if you are returning a single line from a fat arrow function.

```
this.expandButton.addEventListener("click", () => this.expandArticle() );
```